content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md

```diff
@@ -0,0 +1,112 @@
+
+![TasksReplicationTasksAddRemoteDestSCALE](/images/SCALE/RepAddDestinationSCALE.png "Replication with Remote Destination")
+
+{{< hint info >}}
+To use encryption when replicating data click the **Encryption** box. After selecting the box these additional encryption options  become available:
+
+* **Ecryption Key Format** allows the user to choose between a hex (base 16 numeral) or passphrase (alphanumeric) style encryption key. <br>
+* **Store Encryption key in Sending TrueNAS database** allows the user to either store the encryption key in the sending TrueNAS database (box checked) or choose a temporary location for the encryption key that decrypts replicated data (box unchecked)
+{{< /hint >}}
+
+{{< /expand >}}
+
+### Security and Task Name
+
+{{< expand "Security and Task Name" "v" >}}
+{{< hint info >}}
+Using encryption for SSH transfer security is always recommended.
+{{< /hint >}}
+
+In situations where two systems within an absolutely secure network are used for replication, disabling encryption speeds up the transfer.
+However, the data is completely unprotected from eavesdropping.
+
+Choosing **no encryption** for the task is less secure but faster. This method uses common port settings but these can be overriden by switching to the advanced options screen or editing the task after creation.
+
+![TasksReplicationTaskSecuritySCALE](/images/SCALE/RepSecurityTaskSCALE.png "Replication Security and Task Name")
+
+TrueNAS suggests a name based off the selected sources and destination, but this can be overwritten with a custom name.
+{{< /expand >}}
+
+### Define a Schedule and Snapshot Lifetime
+
+{{< expand "Schedule and Lifetime" "v" >}}
+
+Adding a schedule automates the task to run according to your chosen times.
+You can choose between a number of preset schedules or create a custom schedule for when the replication runs.
+Choosing to run the replication once runs the replication immediately after saving the task, but you must manually trigger any additional replications.
+
+Finally, define how long you want to keep snapshots on the destination system.
+We generally recommend defining snapshot lifetime to prevent cluttering the system with obsolete snapshots.
+
+![TasksReplicationTasksScheduleLifeSCALE](/images/SCALE/RepScheduleSCALE.png "Custom Lifetimes")
+{{< /expand >}}
+
+### Starting the Replication
+
+{{< expand "Starting the Replication" "v" >}}
+**Start Replication*** saves the new replication task.
+New tasks are enabled by default and activate according to their schedule or immediately when no schedule is chosen.
+The first time a replication task runs, it takes longer because the snapshots must be copied entirely fresh to the destination.
+
+![TasksReplicationTasksSuccessSCALE](/images/SCALE/RepSuccessSCALE.png "Remote Replication Success")
+
+Later replications run faster, as only the subsequent changes to snapshots are replicated.
+Clicking the task state opens the log for that task.
```

Comment:
```suggestion
Later replications run faster since the task only replicates subsequent changes to snapshots.
Clicking the task state opens the log for that task.
```